addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-connection-service.js

```diff
@@ -178,6 +178,15 @@ class EnvironmentScConnectionService extends Service {
   }
 
   async getRStudioUrl(requestContext, id, connection) {
```

Comment:
What is the difference between environment-url-service getRStudioUrl and this method? One is used for built-in and one for SC?

Reply:
Yes.

Reply:
I havne't added unit tests for the built-in one, since it's deprecated.